app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<% if can? :read, @recipe %>
+<div class="show-recipe-container">
+  <h2><%= @recipe.name %></h2>
+  <div class="recipe-prep-and-public">
+    <p>Preparation time: <%= @recipe.preparation_time %> minutes</p>
+    <div class="recipe-checkbox">
+      Public
+      <label class="switch">
+        <% if @recipe.public %>
+        <input type="checkbox" checked disabled>
+        <% else %>
+        <input type="checkbox" disabled>
+        <%end%>
+        <span class="slider round" checked></span>
+      </label>
+    </div>
+  </div>
+  <p>Cooking time: <%= @recipe.cooking_time %> minutes</p>
+  <p><%= @recipe.description %></p>
+  <% if can? :manage, @recipe %>
+  <a class='recipe-add-ingredient' href="#openAdd">Add food</a>
```

Comment:
- [ ] Make this like as `add ingredient` not `add food` check this wireframe
![fd](https://user-images.githubusercontent.com/44745855/152379715-450fa306-7b59-4a0a-b7fe-652f082a1bc8.png)

Reply:
Done, thanks for the review!